test/SlowTests/MailingList/TestIndexAndTransforSubcollections.cs

```diff
@@ -1,96 +0,0 @@
-using System.Collections.Generic;
-using System.Linq;
-using FastTests;
-using Raven.Client.Documents.Indexes;
-using Raven.Client.Documents.Transformers;
-using Xunit;
-
-namespace SlowTests.MailingList
-{
-    public class TestIndexAndTransforSubcollections : RavenTestBase
-    {
-        [Fact]
-        public void CanTransformMultipleIndexResult()
-        {
-            using (var store = GetDocumentStore())
-            {
-                new IndexSubCollection().Execute(store);
-                new IndexSubCollectionResultTransformer().Execute(store);
-
-                using (var session = store.OpenSession())
-                {
-                    session.Store(new ItemWithSubCollection
-                    {
-                        Name = "MyAggregate",
-                        SubCollection = new[]
-                        {
-                            "SubItem1",
-                            "SubItem2",
-                            "SubItem3"
-                        }
-                    });
-                    session.SaveChanges();
-                }
-
-                using (var session = store.OpenSession())
-                {
-                    var result = session.Query<IndexSubCollectionResult, IndexSubCollection>()
-                        .Customize(customization => customization.WaitForNonStaleResultsAsOfNow()
-                            .SetAllowMultipleIndexEntriesForSameDocumentToResultTransformer(true))
```

Comment:
Restore candidate: in a modified version, due to `SetAllowMultipleIndexEntriesForSameDocumentToResultTransformer` usage? I'm not sure if that's still relevant